apps/manager_app/auth_card.c

```diff
@@ -324,6 +335,9 @@ static void prepare_card_auth_context(auth_card_data_t *auth_card_data) {
            sizeof(auth_card_data->ctx.message),
            UI_TEXT_PLACE_CARD_TILL_BEEP,
            SIGN_SERIAL_BEEP_COUNT(auth_card_data->ctx.pair_card_required));
+
+  memcpy(auth_card_data->ctx.family_id, get_family_id(), FAMILY_ID_SIZE);
+  return MANAGER_TASK_SUCCESS;
 }
 
 static manager_error_code_t handle_sign_card_serial(
```

Comment:
body of `handle_sign_card_serial` and `handle_sign_challenge` is almost same. would be best if the duplication is removed.

Reply:
updated, please check